### DISCUSSION TITLE
Getting a ton of "relative URL without a base"

### DISCUSSION THREAD

**Discussion by atrofimov-sc** [swc-project/swc#8911] [General]
Trying to file an issue, but since there's no repro, it gets closed by bots.
This is the issue: https://github.com/swc-project/swc/issues/8910
I see other people try to file the same issue like here: https://github.com/swc-project/swc/issues/8628 and running into the same problem.

**Comment by atrofimov-sc** [comment]
Downgrading back to `npm install -D @swc/jest@0.2.26 @swc/core@1.3.56` "fixes" the issues until it's fixed in the latest version.
![Screenshot 2024-05-02 at 10 01 47 PM](https://github.com/swc-project/swc/assets/160278133/edb097ac-137e-446d-9058-8a97dc381bdb)

**Comment by kdy1** [comment]
Can you provide a minimal reproduction?

**Comment by kdy1** [comment]
Please file an issue with a valid minimal repro
